src/imdbpie/auth.py

```
@@ -2,84 +2,34 @@
-                canonical_qs,
-                canonical_headers,
-                '',
-                http_request.body,
-            )
-        )
-        return string_to_sign, headers_to_sign
-
-
 def _get_credentials():
     url = '{0}/authentication/credentials/temporary/ios82'.format(BASE_URI)
     response = requests.post(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)